metakb/main.py

```diff
@@ -0,0 +1,61 @@
+"""Main application for FastAPI."""
+from fastapi import FastAPI, Query
+from fastapi.openapi.utils import get_openapi
+from metakb.query import QueryHandler
+from metakb.schemas import SearchService
+from typing import Optional
+
+app = FastAPI(docs_url='/api/v2', openapi_url='/api/v2/openapi.json')
+query = QueryHandler(uri="bolt://localhost:7687",
+                     credentials=("neo4j", "admin"))
+
+
+def custom_openapi():
+    """Generate custom fields for OpenAPI response."""
+    if app.openapi_schema:
+        return app.openapi_schema
+    openapi_schema = get_openapi(
+        title="The VICC Meta-Knowledgebase",
+        version="2.0",
+        description="A search interface for cancer variant interpretations"
+                    " assembled by aggregating and harmonizing across multiple"
+                    " cancer variant interpretation knowledgebases.",
+        routes=app.routes
+    )
+
+    openapi_schema['info']['contact'] = {
+        "name": "VICC",
+        "email": "help@cancervariants.org"
+    }
+    app.openapi_schema = openapi_schema
+    return app.openapi_schema
+
+
+app.openapi = custom_openapi
+search_summary = ("Given variation, disease, therapy, and/or gene, "
+                  "return associated statements and propositions.")
+search_response_description = "A response to a validly-formed query."
+search_description = ("Return statements and propositions associated"
+                      " to the queried concepts.")
+v_description = "Variation to search"
+d_description = "Disease to search"
+t_description = "Therapy to search"
+g_description = "Gene to search"
+s_description = "Statement ID to search"
```

Comment:
Is it worth putting the data model terms (subject, object, etc) in parentheses here?

Reply:
Yes!